packages/bruno-app/src/providers/Hotkeys/index.js

```diff
@@ -144,6 +145,23 @@ export const HotkeysProvider = (props) => {
     };
   }, [setShowNewRequestModal]);
 
+  // close tab hotkey
+  useEffect(() => {
+    Mousetrap.bind(['command+w', 'ctrl+w'], (e) => {
+      dispatch(
+        closeTabs({
+          tabUids: [activeTabUid]
+        })
+      );
+
+      return false; // this stops the event bubbling
+    });
+
+    return () => {
+      Mousetrap.unbind(['command+w', 'ctrl+w']);
+    };
+  }, [activeTabUid, tabs, collections, setShowNewRequestModal]);
```

Comment:
You only need to pass the dependencies to `useEffect` that are being used inside it.
In this case in line 163, we only need to pass `[activeTabUid]`